yaml-tests/src/test/resources/catalog.yamsql

```diff
@@ -62,8 +62,9 @@ test_block:
       - query: select sum(cnt) from (select count(*) as cnt, template_name, template_version from schemas
           group by template_name, template_version having template_name = 'TEST_TEMPLATE_1') as t;
       - explain: "AISCAN(TEMPLATES_COUNT_INDEX [EQUALS promote(@c29 AS STRING)] BY_GROUP -> [_0: KEY:[0], _1: KEY:[1], _2: VALUE:[0]]) | MAP (_._2 AS CNT, _._0 AS TEMPLATE_NAME, _._1 AS TEMPLATE_VERSION) | MAP (_ AS _0) | AGG (sum_l(_._0.CNT) AS _0) | ON EMPTY NULL | MAP (_._0._0 AS _0)"
-      - maxRows: 0 # Disable force continuations because of empty continuation due to: https://github.com/FoundationDB/fdb-record-layer/issues/3206
+      - maxRows: 1
```

Comment:
This one claims to be from #3206, which I think is one that you're currently working on. If that fix isn't actually required, then that's interesting. This may have been miscategorized, or maybe there are interactions with #3206 and #3219 or something

Reply:
yes maybe it was miscategorized?